Updating Product OS sidebar (#11067)
PostHog/posthog.com: contents/docs/getting-started/send-events.mdx

There's no limit to the number of properties an event can have, and your future self will thank you when you need to access a property that initially seemed unecessary.

### Keeping events consistent and reliable

Using custom events also helps keep your tracking consistent and reliable.

When elements of your frontend change, this can affect how autocapture events show up in PostHog.

If you changed the text for your 'Add to cart' button to just 'Add', our autocapure events would begin showing up as `Clicked button with text 'Add'` instead of what they had previously been.

While combining these two events using [actions](/docs/data/actions) to fix this drift is possible, manually tracking these high-value actions with custom events is far more reliable.

### Naming your custom events

We recommend adding custom events for the most important actions in your product that are unlikely to change, such as user sign-ups, purchases, and when features are used. 

While you can name your events however you'd like, we typically recommend using a `[object][verb]` format, where `[object]` is whatever entity the action relates to, and the `[verb]` is the action itself. 

> Some examples of this could include: `project created`, `user signed up`, or `invite sent`.

Reviewer: ```suggestion
Some examples of this include `project created`, `user signed up`, and `invite sent`.
```
